### PR TITLE
Add support for Sklik SSP amp-ad

### PR DESCRIPTION
**Pull request by jandolezal71**
Hi,

please review my PR for:

- ✨ New amp-ad of type `ssp`
- ✅ Tests are also implemented

Thank you

### CODE REVIEW

File: ads/_config.js [hunk 1]
```diff
@@ -523,6 +523,12 @@ const adConfig = jsonConfiguration({
     renderStartImplemented: true,
   },
 
+  'ssp': {
```
Comment (powerivq) [inline ads/_config.js]: Nit: these configs are sorted in alphabetical order. Pls move it to match the file.

File: ads/ssp.js [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+          done(null);
+
+          return;
+        }
+
+        const ssp = global.sssp;
+
+        ssp.config({
+          site:
+            data.site === undefined ? global.context.canonicalUrl : data.site,
```
Comment (powerivq) [inline ads/ssp.js]: Suggest `data.site || global.context.canonicalUrl`.

File: examples/ads.amp.html [hunk 3]
```diff
@@ -1936,6 +1937,15 @@ <h2>Sklik</h2>
       json='{"zoneId":0, "w": 970, "h": 310}'>
   </amp-ad>
 
+  <h2>SSP</h2>
```
Comment (powerivq) [inline examples/ads.amp.html]: Nit: alphabetical order

**Comment by powerivq** [comment]
Need to fix the travis error also.

File: ads/_config.js (hunk 1 above)
Comment (jandolezal71) [reply ads/_config.js]: Sorted alphabeticaly.

File: ads/ssp.js (hunk 2 above)
Comment (jandolezal71) [reply ads/ssp.js]: Done.

File: examples/ads.amp.html (hunk 3 above)
Comment (jandolezal71) [reply examples/ads.amp.html]: Sorted alphabeticaly.

**Comment by jandolezal71** [comment]
> Need to fix the travis error also.

Thanks. Travis is fixed also.

**Comment by jandolezal71** [comment]
@powerivq Hi, any news about this PR? :-) Thank you

**Comment by powerivq** [comment]
@jandolezal71 I have no further comments. Can you rebase to see if the visual test failure could be fixed?

**Comment by jandolezal71** [comment]
> @jandolezal71 I have no further comments. Can you rebase to see if the visual test failure could be fixed?

Yeah, everything pass after rebase.

**Comment by powerivq** [comment]
@jandolezal71 owner's check is failing for some unrelated reason, I'll come back after it is fixed.

**Comment by powerivq** [comment]
@jandolezal71 The issue should be fixed. Can you rebase again? Everything should pass this time.

**Comment by jandolezal71** [comment]
Done

**Pull request merged**